authors/apps/tests/test_comment_edit_history.py

```diff
@@ -0,0 +1,47 @@
+"""
+Test comments app
+"""
+from django.urls import reverse
+from rest_framework import status
+
+from . import post_article, thread, comment
+
+from authors.apps.tests.base import BaseTestCase
+
+
+class TestCommentHistory(BaseTestCase):
+    """
+    This class tests the functions for editing
+    comment and viewing history of edited comment
+    """
+
+    def test_get_all_comments(self):
+        """
+            test to get all edited comments
+
+        """
+        self.user_access()
+        self.posting_article(post_article)
+        slug = self.slugger()
+        url = f'/api/{slug}/comments/'
+        res = self.client.post(url, data=comment, format="json")
```

Comment:
Good work @okellogabrielinnocent, you should consider creating a helper method for creating a comment so that you just call it in your other tests.

Reply:
Otherwise goodwork

Reply:
Thanks, I tried using that but it wasn't recognizing the token from user